src/main/java/com/rcv/Tabulator.java

```diff
@@ -64,17 +64,21 @@
   private BigDecimal winningThreshold;
   // tracks residual surplus from multi-seat contest vote transfers
   private Map<Integer, BigDecimal> roundToResidualSurplus = new HashMap<>();
+  // precinct Ids which may appear in the cast vote records
+  private Set<String> precinctIds;
```

Comment:
Would it be more accurate to refer to these as precinct names instead of IDs? It's a bit confusing now that we're generating a separate set of IDs for them in `ResultsWriter`.

Reply:
Not more accurate, but I agree it would be less confusing.